src/services/notification.ts

```diff
@@ -213,6 +233,16 @@ export class NotificationService {
     }
 
     /**
+     * Sets if the user wants sound when a new message arrives
+     * @param wantsSound
+     */
+    public setWantsSound(wantsSound: boolean): void {
+        this.$log.debug(this.logTag, 'Requesting sound preference change to', wantsSound);
+        this.notificationSound = wantsSound;
+        this.storeSetting(NotificationService.SETTINGS_NOTIFICATION_SOUND, wantsSound.toString());
```

Comment:
I'd probably do an explicit string conversion here instead of relying on `toString`, just to be on the safe side.